science/openmpi/Portfile

```diff
@@ -6,6 +6,7 @@ PortGroup           select 1.0
 PortGroup           compilers 1.0
 PortGroup           muniversal 1.0
 PortGroup           legacysupport 1.0
+PortGroup           obsolete 1.0
```

Comment:
This makes all the subports obsolete. Move this to around line 49, inside the `if {[string first "-devel" $subport] > 0} {` block.

Reply:
Done